### PR TITLE
Prepare for 0.1.0-alpha10 NuGet package

### PR DESCRIPTION
**Pull request by mr-m-coetzee**
A `0.1.0-alpha10` tag should be created.

The tag can be annotated with the following, if desired:
 * [Add Suave web server support]()
 * [Add Markdown core component](https://github.com/plotly/Dash.NET/commit/cd9fe348b7a3e5f7e92ddd513531139c097a5eda)
 * [Add Checklist core component](https://github.com/plotly/Dash.NET/commit/d7860b4e968506d14f694cf927115a17debc7783)
 * [Add Interval core component](https://github.com/plotly/Dash.NET/commit/afa5c245da5291d3a075e0811c9de97067953406)
 * [Add Location core component](https://github.com/plotly/Dash.NET/commit/a65629b79a64ae5d5144386700a60091b5239b4c)
 * [Add Slider core component](https://github.com/plotly/Dash.NET/commit/f440d9f9fa65e02d6415c5534f2e2e9b31e42f30)
 * [Add component generation]()
 * [Auto open some common submodules](https://github.com/plotly/Dash.NET/commit/552ea0ea151abfe8e6495dcb713bff86dbbb47e9)

### CODE REVIEW

File: src/Dash.NET/Dash.NET.fsproj [hunk 1]
```diff
@@ -21,6 +21,32 @@
     <RepositoryType>git</RepositoryType>
     <FsDocsLicenseLink>https://github.com/plotly/Dash.NET/blob/dev/LICENSE</FsDocsLicenseLink>
     <FsDocsReleaseNotesLink>https://github.com/plotly/Dash.NET/blob/dev/RELEASE_NOTES.md</FsDocsReleaseNotesLink>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0.0</FileVersion>
+    <Version>0.1.0-alpha10</Version>
+    <PackageReleaseNotes>[Add Suave web server support]()
```
Comment (kMutagene) [inline src/Dash.NET/Dash.NET.fsproj]: This is not necessary, it gets added via the release target

Comment (kMutagene) [reply src/Dash.NET/Dash.NET.fsproj]: https://github.com/plotly/Dash.NET/blob/f869c1082f15487b180b4bc4436c85bc1dd135ca/build.fsx#L200

Comment (kMutagene) [reply src/Dash.NET/Dash.NET.fsproj]: Same for version.

Comment (mr-m-coetzee) [reply src/Dash.NET/Dash.NET.fsproj]: Thanks. Updated.

**Pull request merged**